internal/ceb/server.go

```diff
@@ -79,7 +80,17 @@ func (ceb *CEB) dialServer(ctx context.Context, cfg *config, isRetry bool) error
 	grpcOpts := []grpc.DialOption{
 		grpc.WithTimeout(5 * time.Second),
 		grpc.WithUnaryInterceptor(protocolversion.UnaryClientInterceptor(protocolversion.Current())),
-		grpc.WithStreamInterceptor(protocolversion.StreamClientInterceptor(protocolversion.Current())),
+		grpc.WithChainStreamInterceptor(
+			protocolversion.StreamClientInterceptor(protocolversion.Current()),
+
+			// Send and receive keepalive messages along grpc streams.
+			// Some loadbalancers (ALBs) don't respect http2 pings.
+			// (https://stackoverflow.com/questions/66818645/http2-ping-frames-over-aws-alb-grpc-keepalive-ping)
+			// This interceptor keeps low-traffic streams active and not timed out.
+			// NOTE(izaak): long-term, we should ensure that all of our
+			// streaming endpoints are robust to disconnect/resume.
+			inlinekeepalive.KeepaliveClientStreamInterceptor(time.Duration(5)*time.Second),
```

Comment:
Nit: I don't know if this is intentional, but you don't need to type convert constants
```suggestion
			inlinekeepalive.KeepaliveClientStreamInterceptor(5*time.Second),
```

Reply:
Ah yup, I started by having this value fed from config, but felt like it wasn't necessary at this point. I think we might get there eventually though, and this will save our newer engineers from a bit of confusion. As far as I know, it's equivalent post-compilation for constants.